### ISSUE TITLE
Omitted words warning?

### ISSUE THREAD

**Issue by Germaniac** [Lchiffon/wordcloud2#56]
Hi, can you make it so that the program shows warnings (or other ways to let me know) when it can't plot certain words because of lack of space? I'm making a wordcloud of hundreds of usernames and want to make sure none of them is missing.

**Comment by lambdamoses** [comment]
Me too, I have the same issue, especially when a few words appear way more often than other words and often those few most frequent words aren't all plotted.

**Comment by dana-and-monsters** [comment]
Agreed with this^. I am having the issue where I have some words that are shown very frequently (e.g. one word shown 200 times while another is shown only 2x), however this is not always that case as I am building it for usage with a shiny app. It would be great if there was a scaling option where words' font size could be specified separately from their number because then I could scale the font size while allowing people to see how many times the word actually occurred in the text.
